src/Unidevel.OpenWeather/OpenWeatherClient.cs

```diff
@@ -80,6 +92,20 @@ private string buildLocationPart(float longitude = float.NaN, float latitude = f
 
         public void Dispose()
         {
```

Comment:
That's fine if you really need it. If you will create local instance of httpClient you wouldn't need Dispose at all.